docs/api/en/objects/LOD.html

```diff
@@ -72,7 +72,7 @@ <h3>[property:array levels]</h3>
 		<h2>Methods</h2>
 		<p>See the base [page:Object3D] class for common methods.</p>
 
-		<h3>[method:null addLevel]( [param:Object3D object], [param:Float distance] )</h3>
+		<h3>[method:LOD addLevel]( [param:Object3D object], [param:Float distance] )</h3>
```

Comment:
minor: in other places we've used `method:this` when the method is chainable, I'd prefer that. 

See https://threejs.org/docs/#api/en/core/Object3D.

Reply:
Updated. Thanks!